### PR TITLE
new(falco): add flux deployment config

### PR DESCRIPTION
**Pull request by incertum**
Add Falco flux deployment config.

@nikimanoledaki first time using Flux. Looking forward to hearing more tips from you and happy to make any necessary adjustments.

Our new dedicated repo contains relevant docs https://github.com/falcosecurity/cncf-green-review-testing

Action Items:
- [ ] Clarify cluster requirements and setup: https://github.com/falcosecurity/cncf-green-review-testing/issues/2
- [ ] Synthetic Workloads for Falco https://github.com/falcosecurity/cncf-green-review-testing/issues/3, also checkout our current setup in https://github.com/falcosecurity/cncf-green-review-testing. This is just to get us going with something. Open to make any adjustments or change the approach altogether.

Tracking:
- https://github.com/falcosecurity/cncf-green-review-testing/issues/4

CC Falco maintainers @leogr @LucaGuerra @maxgio92

### CODE REVIEW

**Comment by rossf7** [comment]
@incertum This looks good from a Flux POV. On the knode requirements I added some feedback here https://github.com/falcosecurity/cncf-green-review-testing/issues/2#issuecomment-1887795831

File: clusters/falco.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# The Falco Authors
```
Comment (nikimanoledaki) [inline clusters/falco.yaml:1]: We changed the repo structure to the following:
```
tree clusters
clusters
├── projects
│   └── falco
│       └── falco-namespace.yaml
├── base
│   ├── kepler-grafana.yaml
│   ├── kepler.yaml
│   └── kube-prometheus-stack.yaml
└── flux-system
    ├── gotk-components.yaml
    ├── gotk-sync.yaml
    └── kustomization.yaml
```
 Could move the config to `clusters/projects/falco` please?

**Review by nikimanoledaki — APPROVED** [review]
Amazing, thank you! A couple of changes then LGTM! 🥳 🎉

**Review by rossf7 — APPROVED** [review]
LGTM 🚀

**Comment by rossf7** [comment]
@incertum we've migrated to the new cluster managed with tofu. So we're ready to merge this.

Could you sign the commits? As there is a problem with the DCO check. Thx!

**Comment by incertum** [comment]
> @incertum we've migrated to the new cluster managed with tofu. So we're ready to merge this.
> 
> Could you sign the commits? As there is a problem with the DCO check. Thx!

I did commit them with sign-off last week, seems all checks have passed. Are there still issues?

**Comment by rossf7** [comment]
> I did commit them with sign-off last week, seems all checks have passed. Are there still issues

@incertum the sign-off looks good but could you [enable GPG so that signed commits are verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) and then rebase please?

**Pull request merged**